solidstart-tanstackquery-tailwind-modules/msw/data.ts

```diff
@@ -0,0 +1,71 @@
+export const readMeResponse = {
+  data: {
+    repository: {
+      readme: {
+        text: 'text',
+      }
+    }
+  }
+};
+
+export const repoInforResponse = {
```

Comment:
why is this data here? It's not part of this ticket

Reply:
I shouldn't have done this.